app/util/remote_runner.tsx

```diff
@@ -53,6 +69,16 @@ export function triggerRemoteRun(invocationModel: InvocationModel, command: stri
     });
 }
 
+// commandWithRemoteRunnerFlags adds useful flags to bazel commands run on a remote runner
+export function commandWithRemoteRunnerFlags(command: string): string {
+  // The buildbuddy_ configs point to the corresponding env that created the workflow
+  // action (Ex. If it was created in dev, points to the dev app)
+  const addlFlags =
+    "--remote_cache_compression --config=buildbuddy_bes_backend --config=buildbuddy_bes_results_url --config=buildbuddy_remote_cache";
```

Comment:
hm I thought we applied the BES flags by default? maybe only the remote cache ones are needed? otherwise lgtm

Reply:
The problem is we have this in our .bazelrc

common --bes_results_url=https://app.buildbuddy.io/invocation/
common --bes_backend=grpcs://remote.buildbuddy.io